CMakeLists.txt

```diff
@@ -153,8 +153,8 @@ set (PSL_LIB_NAME ${_prefix}${_name}${_lib_suffix})
 
 # Get name of shared supplemental library
 if (BUILD_SUPPLEMENTS)
-	get_target_property (_name supplements OUTPUT_NAME)
-	get_target_property (_prefix supplements PREFIX)
+	get_target_property (_name ${GMT_SUPPL_LIB_NAME} OUTPUT_NAME)
+	get_target_property (_prefix ${GMT_SUPPL_LIB_NAME} PREFIX)
 	set (GMT_SUPPL_LIB_NAME ${_prefix}${_name}${CMAKE_SHARED_MODULE_SUFFIX})
```

Comment:
```
set (GMT_SUPPL_LIB_NAME ${_prefix}${_name}${CMAKE_SHARED_MODULE_SUFFIX})
```
This lines changes `GMT_SUPPL_LIB_NAME` to `supplements.so` or `supplements.dll`. We may need two separate names. Why does it work?

Reply:
I dont know, my variable changes does not change that fact, no?

Reply:
GMT_SUPPL_LIB_NAME with the new value (supplements.so or supplements.dll) is used only in `src/config.h.in`. Probably that's the reason why your changes don't break anything. But it would be better to use different variable names to avoid future confusions.

Reply:
Maybe you can fix this since I am not sure what two names.  Also, some simplifications are coming via #3159.

Reply:
OK. Will do it in another PR.

Reply:
Great, thanks. Dinner for me now.